src/Illuminate/Console/Application.php

```diff
@@ -273,6 +273,10 @@ public function resolve($command)
             return null;
         }
 
+        if ($command instanceof Command) {
```

Comment:
@freekmurze Can adjust the PHP Annotations of the `resolve` method, and add a test on this?

Reply:
Annotation adjusted. Like said in the PR description, I didn't find any meaningful tests for this method to build on.